examples/3d-tiles/app.js

```diff
@@ -206,30 +215,31 @@ export default class App extends PureComponent {
     );
   }
 
-  _onTileLoaded(tileHeader) {
-    const {name} = this.state;
-    // cannot parse the center from royalExhibitionBuilding dataset
-    const isRoyal = name === 'royalExhibitionBuilding' && tileHeader.depth === 1;
-    if (tileHeader.depth === 0 || isRoyal) {
-      const {center} = tileHeader.boundingVolume;
-      if (!center) {
-        // eslint-disable-next-line
-        console.warn('center was not pre-calculated for the root tile');
-      } else {
-        scratchLongLat.copy(center);
-        if (isRoyal || name === 'TilesetPoints') {
-          Ellipsoid.WGS84.cartesianToCartographic(center, scratchLongLat);
+  _onTilesetLoaded(tileset) {
+    const root = tileset._root;
+    // TODO: won't be needed after data PR merges, just use root
+    const tileHeader = tileset.isRoyal ? root.children[0] : root;
+    const {center} = tileHeader.boundingVolume;
```

Comment:
For `sphere` and `region` [cases](https://github.com/uber-web/loaders.gl/blob/master/modules/3d-tiles/src/tileset/helpers/bounding-volume.js#L35), `tileHeader.boundingVolume.center` is in cartographic.

Reply:
Only region does carto at this point. I was testing with royal tilesetpoints and some random pointcloud examples. Not sure if there's an example with a region bounding volume to test with.